docs/reference/search/search-your-data/retrievers-overview.asciidoc

```diff
@@ -67,59 +67,131 @@ When using compound retrievers, only the query element is allowed, which enforce
 
 [discrete]
 [[retrievers-overview-example]]
-==== Example
+==== Examples
 
-The following example demonstrates the powerful queries that we can now compose, and how retrievers simplify this process. We can use any combination of retrievers we want, propagating the
-results of a nested retriever to its parent. In this scenario, we'll make use of all 4 (currently) available retrievers, i.e. `standard`, `knn`, `text_similarity_reranker` and `rrf`.
-We'll first combine the results of a `semantic` query using the `standard` retriever, and that of a `knn` search on a dense vector field, using `rrf` to get the top 100 results.
-Finally, we'll then rerank the top-50 results of `rrf` using the `text_similarity_reranker`
+Let's work through some examples to see how we can use and leverage retrievers to build an awesome search experience!
```

Comment:
I'm not sure about the style here in terms of using exclamation marks, adjectives like "awesome" etc. This is our tech docs, not a blog post - I'd expect the tone to be different.

I think having docs team confirmation on this would be good.

Reply:
heheh yeah that's a fair point. Was writing the blog post and the docs at the same time so the tone here has definitely been affected a bit 😅

Reply:
Yup totally, I pointed this out in edits to the examples, but I forgot to edit the intro!

Let's try to keep this concrete and practical, outlining the value of the examples for someone discovering retrievers :)